.github/workflows/build.yaml

```diff
@@ -33,11 +33,41 @@ jobs:
           path: |
             **/build/reports/tests/test/
 
+  android-emulator:
```

Comment:
Would a more descriptive name for this be something like `instrumented-tests`? Seeing `android-emulator` as a job is a bit confusing.

Reply:
Changed to `instrumentation-tests` per LeakCanary: https://github.com/square/leakcanary/blob/main/.github/workflows/main.yml#L28